library/src/interactor_impl.cxx

```diff
@@ -440,6 +360,76 @@ class interactor_impl::internals
   }
 
   //----------------------------------------------------------------------------
+  void TriggerInteractionCommands(const std::string& interaction, const std::string& argsString)
+  {
+    const auto modifiers = [&]()
```

Comment:
I'm not against functional programming, but I'm wondering if it's justified here.  
You could just in `ModifierKeys modifiers` local variable.